### PR TITLE
[FRONTEND][TENSPRFLOW] Add split support

### PR DESCRIPTION
**Pull request by nishi-t**
This PR adds following operators support to tensorflow frontend:

* Split
* SplitV

### CODE REVIEW

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 1]
```diff
@@ -1055,3 +1068,6 @@ def test_forward_rel_ops():
 
     # Relational ops
     test_forward_rel_ops()
+
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Suggest to move under transforms.

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -999,6 +999,19 @@ def test_forward_rel_ops():
     _test_forward_rel_op([t1, t2], math_ops.equal)
     _test_forward_rel_op([t1, t2], math_ops.not_equal)
 
+#######################################################################
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Pls add another test case with Split+Concat.

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 3]
```diff
@@ -1024,8 +1039,8 @@ def test_forward_rel_ops():
 
     # Reductions
     test_forward_argminmax()
-    test_forward_reduce()
-    test_forward_mean()
+    #test_forward_reduce()
+    #test_forward_mean()
```
Comment (ashutoshparkhi) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Any particular reason for commenting them?

Comment (ashutoshparkhi) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: Saw changes in your fix.

**Comment by nishi-t** [comment]
Thank you for your review and approve, but the test case that concat after split currently is not passed. For now, I'll comment out this test case.

**Comment by nishi-t** [comment]
@srkreddy1238 Thaks for the information. I'll check it out and work on the problem of handling outputs.

**Comment by alexeyr** [comment]
I think my pull request #2105 got `SplitV` correct. For your case it should be `tuple(np.cumsum(indices)[:-1])`.

**Comment by nishi-t** [comment]
Sorry, I couldn't work for this PR recently. I'll close my PR.
@alexeyr Thank you for submitting PR to support Split.

**Pull request closed without merging**